en/services/high_latency.md

```diff
@@ -43,7 +45,20 @@ Enum | Description
 
 ## Sequences
 
-A GCS will typically have one primary link that is used for all vehicle communications, but may also have secondary links that it can switch to if needed (which will then become the primary link).
+A GCS will typically have one or more (low latency) links that are used for vehicle communications. Separate to this are any high latency links. The high latency links may be active simultaneous to the low latency links during handover operations.
+
+A typical flight sequence will look like:
+- Vehicle starts up on the ground with low latency links active
+- Ground station(s) download, check and sync all [mission protocol](../services/mission.md) items and [parameter protocol](../services/parameter.md) items over low latency link
+- Vehicle starts mission
+- Whilst still in range of low latency link, enable high latency link and confirm transmission of messages over high latency link.
```

Comment:
QGC only switches to the high latency link if it loses the low latency link. This implies that you start the link and run them in parallel for a while. 
I think more likely this is something you do once to setup the HL link, but you don't need to do it when you have the LL running.